docs/docs/atom-archive/getting-started/sections/atom-basics.md

```diff
@@ -0,0 +1,203 @@
+---
+title: Atom Basics
+---
+
+::: danger STOP
+
+This is an archive of the atom documentation. This is only here for historical
+purposes. This is solely a functional reference of how things used to work.
+Processes in this section may have changed in both functionality and
+presentation since these were originally written. Our updated documentation can
+be found at: <insert link here>
+
+:::
+
+### Atom Basics
```

Comment:
I find it problematic to misuse heading levels for styling purposes (if that's the case). How can a document start with a `<h3>`?